### ISSUE TITLE
sending sms with tier app like kdeconnect don't show

### ISSUE THREAD

**Issue by kevlarcade** [SilenceIM/Silence#551]
### Bug description
I send a sms with kdeconnect it sends correctly but silence doen't show the sms send. I test with the same computer and another phone without silence installed, it works.

### How to reproduce
- Have silence as default sms app
- Use kdeconnect to send a sms

**Actual result:** Silence don't show the sms just sent with kde connect, the sms is corectly sent
**Expected result:** Silence should be able to show me the sms

### Device info
- Device: OnePlus 3
- Android version: 7.0 (OxygenOS version 4.0.3)
- Silence version and build ID: 0.15.2 4992dd7

**Comment by BLeQuerrec** [comment]
Is Silence your default SMS app?

**Comment by kevlarcade** [comment]
Yes I don't have other SMS app (only silence ande another one that request google account, I never activate it)

**Comment by Tonat** [comment]
I think it might be because the sms gets written to the system sms database. I had something similar when using SMS Scheduler SMS Scheduler https://f-droid.org/app/com.github.yeriomin.smsscheduler . There the sms (which send successfully unencryptedly) didn't show up in Silence (set as default) but in the stock sms app. Upon importing the system sms database in Silence in showed up there as well (in correct chat and with right timestamp); just with the side-effect that any other sms in the system database got added as well. 
That's my thought on that, please ignore if unrelated.

**Comment by mihapiha** [comment]
I have the same issue with kdeconnnect and silence. When I click on import messages from settings the messages correctly appear in silence.

**Comment by beta-phenylethylamine** [comment]
Any updates on this/possible workarounds? I would definitely like to have my history be preserved when using KDEConnect to send SMS

**Comment by raspbeguy** [comment]
I would like also this problem to be solved. I use Silence too and it would be neat to be able to respond to SMS from the computer. I understand if that makes a security issue to deal with messages that are expected to be end-to-end encrypted, so if it would be possible to deal with unencrypted conversations at least, that would be great.
